src/main/java/tuteez/model/person/lesson/Lesson.java

```diff
@@ -165,7 +165,7 @@ public static boolean isClashingWithOtherLesson(Lesson firstLs, Lesson secondLs)
      *         {@code false} otherwise.
      */
     public static boolean containsClashes(List<Lesson> lessons) {
```

Comment:
Correct me if I'm wrong, but in textbook Code Quality: Naming, it's recommended to use isXXX for boolean methods? Consider using isClashingWithOtherLessons or something.

On this note, it is also recommended to **not** name Lesson objects as lesson1 (or worse, l1), I'll quote the textbook here: 

> Imagine going to the doctor's and saying "My eye1 is swollen"! Don’t use numbers or case to distinguish names.

Reply:
I believe there is nuance to the method because `isClashingWithOtherLessons` potentially implies that I am checking against pre-existing lessons. But that is not the purpose of this method. This method is purely meant to check if any lessons within a given `List<Lesson>` clashes with one another. Additionally what you said is almost right but isXXX is not really a fixed rule, I have attached a screen shot for easy reference 

![image](https://github.com/user-attachments/assets/672bfbb9-d6ab-4298-9a0f-01d5aac63ae4)

Perhaps `hasClashingLessonsInList` ? What do you think of that

Reply:
Noted on your lesson1 point, I will make that change